src/main/java/org/myrobotlab/service/config/ServoConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.service.config;
+
+public class ServoConfig extends Config {
+
+  public Boolean autoDisable;
+  public String controller;
+  public Double currentOutputPos;
+  public Boolean enabled;
+  public Boolean idleDisabled;  
+  public Integer idleTimeout;
+  public Boolean isBlocking;
+  
+  public String pin;
+  public Double rest;
+  public Double speed;
+
+  public Double sweepMax;
+  public Double sweepMin;
+  
+  public Double targetPos;
```

Comment:
questionable if we care about targetPos

Reply:
It works out to be the value of "last input" ...
not sure - I'm going to take it out, if a "real encoder" existed it should be taking the value when starting there